manual/mingw/tools/chocolateyInstall.ps1

```diff
@@ -1,64 +1,13 @@
-﻿# Default values
-$packageName = 'mingw'
-$packageVersion = '4.8.3'
-$rev = 'rev0'
-$threads = 'posix'
-$exception = 'default' #dwarf is 32bit only, seh is 64bit only, sjlj works with 32 / 64
+﻿. (Join-Path (Split-Path -parent $MyInvocation.MyCommand.Definition) chocolateyInclude.ps1)

```

Comment:
Yup, wanted to make sure - `&` runs the file as a command, which won't have output.

Usually dot sourcing makes commands available. :)